modules/oe_whitelabel_helper/config/install/daterange_compact.format.oe_whitelabel_date_only_short_month.yml

```
@@ -0,0 +1,16 @@
+langcode: en
+status: true
+dependencies: {  }
+id: oe_whitelabel_date_only_short_month
+label: 'Date only, short month'
+default_pattern: 'd M Y'
+default_separator: ' - '
+same_day_start_pattern: 'd M Y'
```

I don't see a leading zero in the pdf for other date formats. So I assume we also don't want it here.
```suggestion
same_day_start_pattern: 'j M Y'
```